src/main/java/app/hero_maker/DataValidator.java

```diff
@@ -0,0 +1,33 @@
+package app.hero_maker;
+
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
+
+import java.util.Arrays;
+
+@Component
+public class DataValidator implements org.springframework.validation.Validator {
```

Comment:
To nie jest 'DataValidator' ta klasa waliduje tylko obiekty "Hero" wiec jets to HeroValidator, przeceiz jak bedziesz robil nastepny ficzer to nie nazwiesz validatora 'DataValidator2', staraj sie zawsze unikac takich ogolnych nazw